pkg/render/render.go

```diff
@@ -247,7 +247,7 @@ func OAuthAppEnv(ctx context.Context, db kclient.Client, oauthAppNames []string,
 
 	for _, integration := range slices.Sorted(maps.Keys(activeIntegrations)) {
```

Comment:
I would expect the render.go to look up all the oauthapp associated with the thread and then merge them with the agent level oauth apps.  This does mean you have to look at the "project thread", not the current thread, so the thread.spec.parentthreadname of the non-project thread.

Reply:
addressed.